src/lib/resource/sections/applicant-when-did-the-crime-stop.js

```diff
@@ -61,8 +61,11 @@ module.exports = {
                 {
                     target: 'p-applicant-select-reasons-for-the-delay-in-making-your-application',
                     cond: [
-                        'dateExceedsTwoYearsFromNow',
-                        '$.answers.p-applicant-when-did-the-crime-stop.q-applicant-when-did-the-crime-stop'
+                        'dateCompare',
+                        '$.answers.p-applicant-when-did-the-crime-start.q-applicant-when-did-the-crime-start',
+                        '>=',
```

Comment:
Is it definitely >= and not >

Reply:
It is comparing  and integer to an integer, so it needs be `>=`.
e.g.

`$.answers.p-applicant...-crime-happen` (1) >= 2 (it .abs() it internally) // false
`$.answers.p-applicant...-crime-happen` (2) >= 2 (it .abs() it internally) // true
`$.answers.p-applicant...-crime-happen` (3) >= 2 (it .abs() it internally) // true

If it was just `>`:

`$.answers.p-applicant...-crime-happen` (1) > 2 (it .abs() it internally) // false
`$.answers.p-applicant...-crime-happen` (2) > 2 (it .abs() it internally) // false
`$.answers.p-applicant...-crime-happen` (3) > 2 (it .abs() it internally) // true


There is a comment on the relevant ticket around what "2 years" actually means. So this logic may need to change.


The "`>=`" logic accounts for a date that is exactly 2 years ago to the day .... and beyond.
The "`>`" logic accounts for beyond exactly 2 years.

If the logic needs to be "exactly 2 years ago or earlier", then `> 1` would suffice.

`$.answers.p-applicant...-crime-happen` (1) > 1 (it .abs() it internally) // false
`$.answers.p-applicant...-crime-happen` (2) > 1 (it .abs() it internally) // tue
`$.answers.p-applicant...-crime-happen` (3) > 1 (it .abs() it internally) // true